_posts/newsletter/2021-09-09-this-month-in-solid.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: newsletter
+title: This Month in Solid 2021-09-09
+permalink: /newsletter/2021-09-09
+tags: [monthly, updates]
+categories: [Updates]
+---
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+**02 September 2021** — As we close out the summer for many, we kick off the upcoming fall months with September’s Solid World.  Solid members are making some of the most exciting and unique applications using Solid. This month we dived into learning all the contributions two developers are making with their applications. 
+
+Ruben Verborgh started the event by updating everyone about how Solid community members adopt the Community Solid Server. Great news: community members support the new server by sharing what they are working on and possible use cases they see. In addition to this, community members are even making new functionalities. For example, one of the community members created a way to generate modules, [here’s the demo of how to do that](https://github.com/FUUbi/solid-css-demo-module). If you want to find a way to contribute, join the Community Solid Server [gitter channel](https://gitter.im/solid/community-server), share, create, or learn what others are doing. 
+
+Next, we had Oz Olivo, who gave a rundown on the Enterprise Solid Sever and Inrupt’s roadmap. For many of us, we are preparing to fill up the coming months with exciting work. Inrupt is no exclusion to this as they plan to get Solid to the forefront of mainstream initiatives. Hopefully, the projects driving this can share their development in a future Solid World. It’s a way for others outside of the community to see what we all see, the promise of Solid. Addedly they are working on getting their V1.2 release out of the door. The beta version will be available for testing next month, including their libraries. So be on the lookout for how you can assist in their beta.
```

Comment:
Today I learned the term "addedly" :)

Reply:
hehe wait is it a word?

Reply:
Yeah I looked it up! https://en.wiktionary.org/wiki/addedly

Reply:
Haha ayoooo look at me knowing words I didn't know haha